packages/agoric-cli/src/commands/wallet.js

```diff
@@ -37,12 +37,28 @@ export const makeWalletCommand = async () => {
       normalizeAddress,
     )
     .option('--spend', 'confirm you want to spend')
+    .option('--home [dir]', 'agd application home directory')
+    .option(
+      '--keyring-backend [os|file|test]',
+      'keyring\'s backend (os|file|test) (default "os")',
+    )
+    .option('--dry-run', 'spit out the command instead of running it')
```

Comment:
doesn't `spend` option handle this case?

Reply:
no; if you omit `--spend`, it doesn't spit out the command.
Oh... I suppose it could.

Reply:
yeah I think that would be better. the message could say ~"re-run this with --spend or run the following command:"